src/applications/avs/tests/utils/index.test.unit.spec.js

```diff
@@ -0,0 +1,137 @@
+import { assert, expect } from 'chai';
```

Comment:
BIGGER THOUGHT THAN THIS PR: Do we like tests for things in the same directory or in a single directory like this is?  We can theory-craft  around this at one of our engineering meetings, but this is totally project/personal preference.

Reply:
I like the idea of discussing this more, lots of projects have a mix and I decided to basically flip a coin and do it only one way here.